x/foundation/keeper/keys.go

```diff
@@ -13,15 +13,15 @@ var (
 	paramsKey         = []byte{0x00}
 	foundationInfoKey = []byte{0x01}
 
-	// treasuryKey = []byte{0x10}
+	memberKeyPrefix          = []byte{0x10}
+	previousProposalIDKey    = []byte{0x11}
+	proposalKeyPrefix        = []byte{0x12}
+	proposalByVPEndKeyPrefix = []byte{0x13}
+	voteKeyPrefix            = []byte{0x14}
```

Comment:
Why this properties are changed?

Reply:
Because we don't need the validator auth key anymore.